packages/ERTP/test/swingsetTests/basicFunctionality/bootstrap.js

```diff
@@ -1,5 +1,5 @@
 import { E } from '@endo/eventual-send';
-import { assert, details as X } from '@agoric/assert';
+const { details: X } = assert;
```

Comment:
Not sure why lint is correctly flagging some of these as needing to be moved to after the imports, but is not flagging others, such as this one. In any case, all of these need to be after the imports.